src/Microsoft.Azure.SignalR.Management/Microsoft.Azure.SignalR.Management.csproj

```diff
@@ -46,4 +43,11 @@
   <ItemGroup Condition=" '$(TargetFramework)' == 'netstandard2.0'">
     <PackageReference Include="Microsoft.AspNetCore.SignalR" Version="$(MicrosoftAspNetCoreSignalRPackageVersion)" />
   </ItemGroup>
+
+  <!-- Specifying signing for multiple files in a package. -->
+  <ItemGroup>
```

Comment:
move to `directory.Build.props`?

Reply:
I thought different packages have different dlls. Looks move the common Directory.Build.props also works. Nice catch.